tests/integrationTests/inputs/shockTube2Gas2D.yaml

```diff
@@ -0,0 +1,90 @@
+---
+environment:
+  title: SOD_2Gas_6x6_ArAir
+  tagDirectory: true
+arguments: {}
+timestepper:
+  name: theMainTimeStepper
+  arguments:
+    ts_type: rk
+    ts_max_time: 0.01
```

Comment:
You dt is of fixed size and adapt is off.  This may be ok, but you have to make sure you don't step over the cfl constraint.